store/index.js

```diff
@@ -164,6 +168,18 @@ export const mutations = {
 
         state.portal = allocation
         state.room.portal = allocation
+
+        if(allocation.status === 'creating' || allocation.status === 'starting' || allocation.status === 'open')
```

Comment:
Oh, there's also this. This wouldn't be required at all (leave it only if status is `open`) if the other change I mentioned earlier is done, since it'd not show it up anyway. Although, wouldn't it also be good to do it if status isn't `in-queue`?

Reply:
But actually, thinking about the last one, since most like QUEUE_UPDATE might be sent before, it would not make it work.

Reply:
This is partially a workaround to the above mentioned and just removing un-needed state.